packages/bundler/src/BundlerCollectorTracer.ts

```diff
@@ -55,6 +55,7 @@ export interface TopLevelCallInfo {
   opcodes: { [opcode: string]: number }
   access: { [address: string]: AccessInfo }
   contractSize: { [addr: string]: ContractSizeInfo }
+  extCodeAccessInfo: { [addr: string]: string }
```

Comment:
why add another struct? that was what the "opcode" meant in the "contractSize" dictionary.

Reply:
We agreed to only store the last "access" to the contract code in the `contractSize` field. This means the attacker could make something like ```int size = extcodesize(entryPoint); entryPoint.depositTo(0); ``` and the `contractSize` would have `{ opcode: "CALL" }` and completely missing the `EXTCODESIZE`.

Do you think I should address this issue in a different way?